packages/bpk-component-link/readme.md

```diff
@@ -50,3 +50,7 @@ For both BpkLink and BpkLinkButton the following theme attributes are required.
 * `linkHoverColor`
 * `linkActiveColor`
 * `linkVisitedColor`
+* `linkAlternateColor`
```

Comment:
Add these under different headings since they are exported separately. See the buttons README.md for an example

Reply:
done